config/init.php

```diff
@@ -1,3 +1,6 @@
 <?php
 include('config/config.php');
 include('include/db_query.php'); //this should happen right after config so other functions have access to the database
+include('include/header.php');
+include('include/footer.php');
```

Comment:
This isn't a problem at all, but I just want to point out that you don't need a separate file for each function. the Header and Footer functions are similar enough that they can probably go in the same file, but this is fine too.